ppr-ui/src/composables/useRegistration.ts

```diff
@@ -44,10 +44,10 @@ export const useRegistration = (setSort: RegistrationSortIF) => {
 
   const getStatusDescription = (status: APIStatusTypes | MhApiStatusTypes,
     isChild: boolean, isPpr: boolean, isDraft: boolean): string => {
-    if (!status) return UIStatusTypes.DRAFT
-    if (status === MhApiStatusTypes.FROZEN) return MhUIStatusTypes.ACTIVE
     if (isChild && isDraft) return UIStatusTypes.DRAFT
     if (isChild) return ''
+    if (!status) return UIStatusTypes.DRAFT
```

Comment:
Any specific reason for changing this order?

Reply:
The unchanged order will lead to 'active' status is displayed in the child rows.

Reply:
Is this part the this ticker or another? Not sure how the order (and active status for child row) is related to new Ownership radio component?

Reply:
@RuoxuanPengBC

Reply:
Is PR is fixing multiple tickets, it should be mentioned in the Description.